website/content/v1.3/kubernetes-guides/configuration/storage.md

```diff
@@ -107,6 +107,79 @@ talosctl -n <node ip> service kubelet restart
 
 Continue setting up [Mayastor](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor) using the official documentation.
 
+### Piraeus / LINSTOR
+
+* [Piraeus-Operator](https://piraeus.io/)
+* [LINSTOR](https://linbit.com/drbd/)
+* [DRBD Extension](https://github.com/siderolabs/extensions#storage)
+
+<!-- markdownlint-disable MD007 -->
+<!-- markdownlint-disable MD032 -->
+<!-- markdownlint-disable MD025 -->
+
+{{< tabpane lang="yaml" >}}
+{{< tab header="Strategic merge patch" >}}
+machine:
+  install:
+    extensions:
+      - image: ghcr.io/siderolabs/drbd:9.2.0-v1.3.0
+  kernel:
+    modules:
+      - name: drbd
+      - name: drbd_transport_tcp
+{{< /tab >}}
+{{< tab header="JSON patch" >}}
+- op: add
+  path: /machine/install/extensions/-
+  value:
+    - image: ghcr.io/siderolabs/drbd:9.2.0-v1.3.0
+- op: add
+  path: /machine/kernel/modules/-
+  value:
+    - name: drbd
+    - name: drbd_transport_tcp
+{{< /tab >}}
+{{< /tabpane >}}
+
+```sh
+# Apply config update on node
+talosctl apply -f install-drbd.yml -n node01
+
+# Create dedicated namespace for piraeus
+kubectl create namespace piraeus
+
+# Install Piraeus with Helm chart from GitHub repository
+git clone https://github.com/piraeusdatastore/piraeus-operator.git
+cd piraeus-operator
+helm install piraeus-op ./charts/piraeus \
+  --namespace piraeus \
+  --set operator.satelliteSet.kernelModuleInjectionMode=DepsOnly \
+  --set etcd.enabled=false \
+  --set controller.dbConnectionURL=k8s
+
+# Create device pool with  on node01
+kubectl linstor physical-storage create-device-pool --pool-name nvme_lvm_pool LVM node01 /dev/nvme0n1 --storage-pool nvme_pool
```

Comment:
`kubectl linstor` is not supported by default.
Could you please add the [required steps](https://github.com/piraeusdatastore/kubectl-linstor#requirements--installation) to the instructions?

Reply:
in addition could also recommend a `krew` install using `kubectl krew install linstor` and point to krew [install instructions](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)

Reply:
Added!